kubernetes/charts/edge-kubernetes/templates/iotedged-deployment.yaml

```diff
@@ -26,6 +26,13 @@ spec:
           imagePullPolicy: {{ .Values.iotedged.image.pullPolicy }}
           command: ["/app/iotedged"]
           args: ["-c", "/etc/iotedged/config.yaml"]
+          resources:
+            limits:
+              memory: {{ .Values.iotedged.data.limits.memoryLimit | quote }}
+              cpu: {{ .Values.iotedged.data.limits.cpuLimit | quote }}
+            requests:
+              memory: {{ .Values.iotedged.data.limits.memoryLimit | quote }}
+              cpu: {{ .Values.iotedged.data.limits.cpuLimit | quote }}
```

Comment:
Is it intentionally to have the resource limits == the resources requests? I would suggest to distinguish them to: 
```
.Values.iotedged.data.limits.memory
.Values.iotedged.data.limits.cpu
```

```
.Values.iotedged.data.requests.memory
.Values.iotedged.data.requests.cpu
```

Reply:
The feature request was specifically to set QoS to be Guaranteed, so intent is to set request==limits. And I wanted to limit the number of knobs, but maybe setting both will be useful.

Reply:
Reworked this so resources are passed as defined in Helm chart to appropriate places. This allows the user to set or clear resources as appropriate.  (Default values have not changed.)